README.md

```diff
@@ -13,13 +20,22 @@ Hugging Face handles SSL encryption automatically.
 Deploying in another environment may require additional configuration; previous configurations are discussed in [Prior Configurations](https://github.com/Imageomics/Andromeda/wiki/Prior-Configurations).
 
 ## Development
-You may use the Docker image for testing during development as described above, but it is not required.
 
 To run the website locally without using Docker requires two terminal sessions.
-1. Python Flask Backend API Server 
-2. Frontend nodejs/react development server
-
-For instructions on buliding and running the Docker container see [Developing with Docker](https://github.com/Imageomics/Andromeda/wiki/Developing-with-Docker).
+1. Python Flask Backend API Server: navigate to the `andromeda` folder and run
```

Comment:
I'm wondering whether we shouldn't simply defer to the subsequently mentioned subfolder README for all executable instructions, to prevent these from accidentally diverging in the future (and to prevent unintended mistakes, such as implicitly disabling CORS when that wasn't expected).

Reply:
I added this into the main README, since otherwise you have to go to two separate subfolder READMEs to get instructions for running locally, which can complicate what is otherwise a simple process.